src/Core/Localization/CLDR/TerritoryData.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Localization\CLDR;
+
+/**
+ * The TerritoryData class is the exact representation of Territory's data structure inside CLDR xml data files.
+ */
+class TerritoryData
+{
+    /**
+     * Alphabetic ISO 4217 territory code.
+     *
+     * @var string
+     */
+    protected $isoCode;
```

Comment:
All these fields are nullable since in the override function you check if they are null
So it needs to be refelcted in all the PHPDoc and in the return types `?string`

Reply:
Fixed